src/utils/openAIcall.js

```diff
@@ -0,0 +1,12 @@
+import OpenAI from "openai";
+
+const openai = new OpenAI({ apiKey: process.env.REACT_APP_OPENAI_API_KEY, dangerouslyAllowBrowser: true });
+
+export async function getResponseContent(messages, model, temperature) {
```

Comment:
I think passing in a object instead of 3 separate arguments is better. So the dev doesn't get errors from mixing up the order of the arguments.

Since this call has only one real source of error, you can put the try catch block here. Then you don't need to always have it in the files with rendered pages.